lib/ancestry/instance_methods.rb

```diff
@@ -17,13 +17,7 @@ def update_descendants_with_new_ancestry
           # ... replace old ancestry with new ancestry
           descendant.without_ancestry_callbacks do
             descendant.update_attribute(
-              self.ancestry_base_class.ancestry_column,
-              descendant.read_attribute(descendant.class.ancestry_column).gsub(
-                # child_ancestry_was
-                /^#{self.child_ancestry}/,
-                # future child_ancestry
-                if ancestors? then "#{read_attribute self.class.ancestry_column }/#{id}" else id.to_s end
-              )
+              :ancestor_ids, path_ids + (descendant.ancestor_ids - path_ids_was)
```

Comment:
Is there a reason this needs to stay multi-line now?  I guess to stay under a character count, so I guess that is cool.

Possibly consider doing:

```ruby
updated_ids = path_ids + (descendant.ancestor_ids - path_ids_was)
descendant.update_attribute(:ancestor_ids, updated_ids)
```

But that is just how I tend to organize code.

Reply:
I'm doing this change right now - thanks